<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWin</class>
 <widget class="QWidget" name="ClientMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientMainWin</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,11">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>头像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ChatTBtn">
        <property name="text">
         <string>聊天</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ContactsTbtn">
        <property name="text">
         <string>联系
人</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="UserInfoTBtn">
        <property name="text">
         <string>个人
信息</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>310</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="PendingTBtn">
        <property name="text">
         <string>待定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
